frontend/src/metabase/query_builder/components/QueryModeButton.jsx

```diff
@@ -59,7 +59,7 @@ export default class QueryModeButton extends Component {
                     </span>
                 </Tooltip>
 
-                <Modal medium backdropClassName="Modal-backdrop--dark" isOpen={this.state.isOpen} onClose={() => this.setState({isOpen: false})}>
+                <Modal medium backdropClassName="Modal-backdrop-dark" isOpen={this.state.isOpen} onClose={() => this.setState({isOpen: false})}>
```

Comment:
Following "BEM" this was actually correct and it's the `.Modal-backdrop-dark` selector that should be changed to `.Modal-backdrop--dark`